Sources/Reachability/NetworkReachability.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  NetworkReachability.swift
+//  Cara
+//
+//  Created by Dylan Gyesbreghs on 02/01/2019.
+//
+
+import Connectivity
+
+open class NetworkReachability {
+    
+    // MARK: - Public Properties
+    public typealias Listener = (Connectivity.Status) -> Void
+    
+    public var listener: Listener?
+    
+    public var isPollingEnabled: Bool {
+        set {
+            connectivy.isPollingEnabled = newValue
+            connectivy.startNotifier()
+        }
+        get {
+            return connectivy.isPollingEnabled
+        }
+    }
+    
+    public var isConnected: Bool {
+        return connectivy.isConnected
+    }
+    
+    public var isConnectedViaCellular: Bool {
+        return connectivy.isConnectedViaCellular
+    }
+    
+    public var isConnectedViaWiFi: Bool {
+        return connectivy.isConnectedViaWiFi
+    }
+    
+    public var isConnectedViaCellularWithoutInternet: Bool {
+        return connectivy.isConnectedViaCellularWithoutInternet
+    }
+    
+    public var isConnectedViaWiFiWithoutInternet: Bool {
+        return connectivy.isConnectedViaWiFiWithoutInternet
+    }
+    
+    // MARK: - Internal Properties
+    private lazy var connectivy: Connectivity = {
```

Comment:
`private lazy var connectivity: Connectivity` instead of `connectivy`?

Reply:
Modified the implementation of this property.